### PR TITLE
sol-network: Fix API names

### PR DESCRIPTION
**Pull request by amandawrcoelho**
issue #1716

Signed-off-by: Amanda Coelho <amada.winder.ribeiro.coelho@intel.com>

### CODE REVIEW

**Comment by amandawrcoelho** [comment]
@ceolin done!

File: src/lib/comms/include/sol-network.h [hunk 1]
```diff
@@ -285,18 +285,18 @@ bool sol_network_subscribe_events(void (*cb)(void *data, const struct sol_networ
...
  */
-bool sol_network_unsubscribe_events(void (*cb)(void *data, const struct sol_network_link *link,
+int sol_network_unsubscribe_events(void (*cb)(void *data, const struct sol_network_link *link,
     enum sol_network_event event),
     const void *data);
 /**
  * @brief Retrieve the available network links on system.
  *
- * @return A vector containing the available links @see sol_network_link
+ * @return A vector containing the available links @see sol_network_link, in case of error @return -EFAULT
```
Comment (glima) [inline src/lib/comms/include/sol-network.h]: My doubt still valid here (see comment on previous PR)

Comment (ceolin) [inline src/lib/comms/include/sol-network.h]: it's not, she has already changed it.

Comment (ceolin) [inline src/lib/comms/include/sol-network.h]: I'll push her commit with this fixed. Thank you

**Comment by ibriano** [comment]
Seems fine to me

**Pull request closed without merging**